docs/patterns/existing-eks-observability-accelerators/existing-eks-opensource-observability.md

```diff
@@ -156,7 +156,8 @@ If you need Java observability you can instead use:
         }
       ]
     },
-    "java.pattern.enabled": true
+    "java.pattern.enabled": true,
+    "nginx.pattern.enabled": true
```

Comment:
Remove this because you have separate documentation.

Reply:
I thought that I can add this because java.pattern.enabled was already existed. Then can I remove both enable code or just nginx?

Reply:
just remove the line `"nginx.pattern.enabled": true` 160 from above.